README.md

```diff
@@ -15,6 +15,7 @@ brew install spacelift-io/spacelift/spacectl
 ### Arch linux
 
 Install [`spacectl-bin`](https://aur.archlinux.org/packages/spacectl-bin) from the Arch User Repository ([AUR](https://aur.archlinux.org/)):
+
```

Comment:
One very small thing, can you add the windows package install command to the readme file? 🙏